README.md

```diff
@@ -458,31 +455,35 @@ Now import the owner private key generated previously in the wallet:
 ```
 
 <a name="uploadsmartcontract"></a>
-### Upload sample contract to blockchain 
+### Upload sample "currency" contract to blockchain 
 
-Before uploading a contract, you can verify that there is no current contract:
+Before uploading a contract, we will verify that there is no current contract:
 
 ```bash
 ./eosc get code currency 
 code hash: 0000000000000000000000000000000000000000000000000000000000000000
 ```
 
-With an account for a contract created, you can upload a sample contract:
+With an account for a contract created, we will upload a sample contract:
```

Comment:
Same as above. we -> you

Reply:
`you can` is passive language and should not be employed in a walkthrough or tutorial, as it is disengaging. We are walking through the reader with specific steps, not giving them an option.

Reply:
changed to utility language